### DISCUSSION TITLE
Should I interpret this installation error as Ubuntu 18.04 no longer being supported?

### DISCUSSION THREAD

**Discussion by anohren** [lutris/lutris#5104] [Q&A]
Apt was complaining that the Lutris PPA (http://ppa.launchpad.net/lutris-team/lutris/ubuntu) no longer had a release file for Ubuntu 18.04, so I thought I'd find out whether I could even run the latest Lutris by installing it from other sources (can't really update Ubuntu since I use a 32 bit machine to run old games).

1. Searched and found https://lutris.net/downloads which pointed me to .deb files on Github Releases for Ubuntu.
2. Tried installing the latest .deb
3. `Error: Dependency is not satisfiable: vulkan-tools` -- seems like the `vulkan-utils` package but with a changed name (?)

The closest I got to system requirements were "Lutris is compatible with any up-to-date Linux system" on the downloads page, which doesn't strictly exclude older versions like Ubuntu 18.04, although in this case the dependencies seem to do their part to that end. I wanted to double check that I'm right in concluding that 18.04 is no longer supported?

If not, I suppose the last resort for 32 bit systems will have to be Debian.

**Comment by strycore** [comment]
No, Ubuntu 18.04 is longer supported. It's going to be 6 years old pretty soon...

Outside of the vulkan-tools dependency, you might run into issues using an older version of Python. You could eventually experiment running Lutris from the git source and see if everything works ok.

I've installed Lutris 0.5.14 on Twister OS 32 bit which is Debian 10 based without issue.

> If not, I suppose the last resort for 32 bit systems will have to be Debian.

I would *really* recommend that, yes.

**Reply by anohren** [reply]
Thank you, nice of you to drop by with the answer, and the suggestions. 

18.04 has been going strong simply by providing Kodi v.17, which is most likely the latest version this little machine will ever be able to run. 

Then I'm going to have to check out the 32 bit improvements you've made (especially removing `unzip` from installers) in a recent Debian instead, and perhaps compile Kodi 17 myself.
